lib/KDropdownMenu.vue

```diff
@@ -175,19 +175,36 @@
         const popoverIsOpen = popover.clientWidth > 0 && popover.clientHeight > 0;
         // set current element and its siblings
         let focusedElement = document.activeElement;
-        let sibling = focusedElement.nextElementSibling;
-        let prevSibling = focusedElement.previousElementSibling;
+        // let sibling = focusedElement.nextElementSibling;
+        // let prevSibling = focusedElement.previousElementSibling;
+
+        //function to check if the sibling is divider
+        const isDivider = element => {
+          return element && element.classList.contains('is-divider');
```

Comment:
This is a good way to determine if the element is a divider, but is a little specific. If anything changes and in the future we get rid of this specific class. This would break this condition.

A more general solution could be to ask if the element has a tabIndex >= 0, so this means it is focusable and with this, we can skip the dividers.

Reply:
Oh right, that seems more logical to skip any other element [like divider] in a more generalized way. Thanks @AlexVelezLl .